packages/drawer/index.js

```diff
@@ -51,16 +59,18 @@ class Drawer extends React.Component {
       this.foundation_ = new MDCModalDrawerFoundation(this.adapter);
       this.foundation_.init();
     }
-
-    if (open && this.foundation_) {
-      this.foundation_.open();
-    }
   }
 
   componentDidUpdate(prevProps) {
     const {dismissible, modal, open} = this.props;
+    const changedToModal = prevProps.modal !== this.props.modal;
+    const changedToDismissible = prevProps.dismissible !== this.props.dismissible;
     if (!(dismissible || modal)) return;
 
+    if ((changedToModal || changedToDismissible) && !this.foundation_) {
```

Comment:
```suggestion
    if (!this.foundation_) {
```
If I'm understanding this right, it doesn't matter whether it was changed to modal/dismissible or if it was already modal/dismissible, but as long as it's modal/dismissible and doesn't have a foundation then we should initialize it, right?

Reply:
Modal has a different foundation from dismissible. So actually this might be incorrect